### PR TITLE
Add EU endpoint

### PR DESCRIPTION
**Pull request by dabcoder**
As `dogwrap` does not currently have an option to choose the site.

### CODE REVIEW

File: datadog/dogshell/wrap.py [hunk 1]
```diff
@@ -310,7 +312,12 @@ def main():
         options.sigterm_timeout, options.sigkill_timeout,
         options.proc_poll_interval, options.buffer_outs)
 
-    initialize(api_key=options.api_key)
+    if options.site.lower() == 'eu':
```
Comment (jd) [inline datadog/dogshell/wrap.py]: `lower()` should not be needed if `choices` is enforced. :)

File: datadog/dogshell/wrap.py [hunk 2]
```diff
@@ -253,6 +253,8 @@ def parse_options(raw_args=None):
 as it should appear on your Datadog stream")
     parser.add_option('-k', '--api_key', action='store', type='string',
                       help="your DataDog API Key")
+    parser.add_option('-s', '--site', action='store', type='choice', default='us', choices=['us', 'eu'], help="The site \
```
Comment (zippolyte) [inline datadog/dogshell/wrap.py]: Hmm, for this I would have kept consistency with the agent config and gave the choices `datadoghq.eu` or `datadoghq.com`

Comment (dabcoder) [reply datadog/dogshell/wrap.py]: So your suggested change would be `default='datadoghq.com', choices=['datadoghq.com', 'datadoghq.eu']`?

Comment (jd) [reply datadog/dogshell/wrap.py]: Maybe support both syntaxes?
I mean, it's long to type the long hostname on a CLI.

**Pull request merged**